[Clarabel.cpp] Enable the SDP feature by default

Reasoning for not upstreaming this patch: changing an option, which is
our prerogative but not relevant for upstream.

--- rust_wrapper/Cargo.toml
+++ rust_wrapper/Cargo.toml
@@ -14,6 +14,7 @@ opt-level = 3
 
 [features]
 # Define features for SDP support in Clarabel.rs
+default = ["sdp"]
 sdp = ["clarabel/sdp"]
 sdp-accelerate = ["sdp", "clarabel/sdp-accelerate"]
 sdp-netlib     = ["sdp", "clarabel/sdp-netlib"]
